src/pages/FooterPages/ContributorPage/index.jsx

```diff
@@ -70,7 +70,7 @@ function Contributor() {
       <div className="search-container">
         <input
           type="text"
-          className="search-input"
+          className="search-input search-Contributor"
```

Comment:
you should add this class's css in index.css of contributor page only not in navbar's index.css